### DISCUSSION TITLE
I dont want to send my activity to Newsletter("For You") and "Following" tab

### DISCUSSION THREAD

**Discussion by WilliamLAS** [community/community#43098] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I dont want people to know my activity i want to be private as hell. As you can see, everyone can see my activity if they following me. Even if i go into a repository. If any setting exists for that, i dont know because nothing says about this topic.

**Comment by jfullstackdev** [comment]
you can set it 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/setting-your-profile-to-private
